### PR TITLE
🌱 Update types.go

### PR DESCRIPTION
**Pull request by Sayanta66**
**What this PR does / why we need it**:

Added spec description and fixes.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**TODOs**:

- [ ] squash commits
- [x] include documentation
- [ ] add unit tests

### CODE REVIEW

**Review by janiskemper — CHANGES_REQUESTED** [review]
always start comments with the name of the property. For example
"EnableIPv4 ...."

File: api/v1beta1/types.go [hunk 1]
```diff
@@ -145,7 +148,7 @@ type LoadBalancerSpec struct {
 	// +kubebuilder:default=lb11
 	Type string `json:"type,omitempty"`
 
-	// API Server port. It must be valid ports range (1-65535). If omitted, default value is 6443.
+	// API Server port. It must be a valid ports range (1-65535). If omitted, the default value is 6443.
```
Comment (janiskemper) [inline api/v1beta1/types.go:151]: start with Port

**Review by janiskemper — CHANGES_REQUESTED** [review]

**Review by janiskemper — CHANGES_REQUESTED** [review]

File: api/v1beta1/types.go [hunk 2]
```diff
@@ -196,20 +209,22 @@ type LoadBalancerTarget struct {
...
 	Enabled bool `json:"enabled"`
 
-	// CIDRBlock defines the cidrBlock of the HCloud Network. A Subnet is required.
+	// CIDRBlock defines the cidrBlock of the HCloud Network.
 	// +kubebuilder:default="10.0.0.0/16"
 	// +optional
 	CIDRBlock string `json:"cidrBlock,omitempty"`
 
 	// SubnetCIDRBlock defines the cidrBlock for the subnet of the HCloud Network.
+	// Note: A subnet is required.
```
[the remark this answers is not in the document]
Comment (Sayanta66) [reply api/v1beta1/types.go:221]: What do you mean by this?

[the remark this answers is not in the document]
Comment (janiskemper) [reply api/v1beta1/types.go:221]: there is a default value that is used usually! you can mention this, as you did in other cases as well

**Review by janiskemper — APPROVED** [review]

**Pull request merged**